### DISCUSSION TITLE
権利侵害の削除要請

### DISCUSSION THREAD

**Discussion by ken-okabe** [increments/qiita-discussions#92] [ヘルプ] [answered]
Qiitaでは、 プロバイダ責任制限法に基づく、送信停止措置対応はしていますか？

**Comment by kskwtnk** [comment, marked as answer]
@stken2050 

ご質問ありがとうございます！

送信停止措置対応は、個別の事案ごとに現行法に則って検討するのと、法律に従って判断し、必要な対応を行います！

申請を受けたものすべてについて送信停止措置対応を行っているわけではありませんので、その点はご理解いただけると幸いです 🙏

**Reply by ken-okabe** [reply]
ご回答どうもありがとうございます。

support@qiita.com
にプロバイダ責任制限法に基づく、権利侵害の書き込みについて送信停止措置対応を正式な法的フォーマットにて送信しましたが、
お問い合わせを承りました: 権利侵害情報の削除要請
お問い合わせいただきありがとうございます。
内容を確認の上、弊社サポートチームが折り返しご連絡いたします。

というメールが来たまま、まる一日経過しているので、どうなっているのか確認したいです。
「折り返し連絡いたします。」ということですので、仮に判断がどうであろうとも、無視されたまま連絡がない、ということはありえないという理解で構いませんか？法的フォーマットに則った要請をサポートメールに出して、連絡があると言いながら無いというのは社会通念上ちょっと考えにくいですので。
ちなみにHatenaその他の運営では通常このような法的フォーマットのメールには平日営業中では24時間以内にだいたい返答があります。

**Reply by kskwtnk** [reply]
@stken2050 

お待たせしてしまい申し訳ありません！

ご連絡の確認はしており、追ってご返信いたしますので今しばらくお待ちください 🙇

**Reply by ken-okabe** [reply]
そうですか。それならば良いです。
